### ISSUE TITLE
(Maybe) Add more CBT features

### ISSUE THREAD

**Issue by EvgeniiaVak** [Romex91/self-help-sheets#9]
[The feeling good](https://feelinggood.com/) approach has additional features. E.g. there is a [spreadsheet](https://feelinggood.com/wp-content/uploads/2020/08/Daily-Mood-journal-with-goal-1-a-with-Do-Not-Copy.pdf) that features some more columns. To my opinion it's fine to go without `%` because emoji are a lot better at this, but I would really like to add the **distortion category column** ( 🤔 although on the second thought, it might contribute to not approving yourself... something like "oh, I did labeling, how bad that I'm doing labeling all the time, look at all this labeling everyday", etc).

Another feature that would be nice to have is **prediction charts** on a separate page - when you think about a future event, estimate how bad/good it'll be attending this event and then on the right columns fill in just right after this event how it actually went.

**Comment by Romex91** [comment]
You can customize your questions to have both features without extra columns.
E.G. you can add "How do you predict you will feeling?" to the left column and "Was your prediction true?" to the right.

I understand that it is not as _pretty_ as if it was a feature, but I thought maybe it is enough. What do you think?

I agree that custom-columns-number feature will be a nice СВИСТЕЛКА/ПЕРДЕЛКА, but is it a must-have?

**Comment by EvgeniiaVak** [comment]
@Romex91 No, it's absolutely not a must-have (as all the ideas in this issue, hence the `(maybe)`), and it's not even as desirable (at least for me) as the second one - the predictions table.

**Comment by EvgeniiaVak** [comment]
@Romex91 ah, I reread your question, (previously I thought you were talking about the distortions category column).

I would really like this prediction table to be on a separate page and have some numbers columns. And again - not a must-have.

**Comment by EvgeniiaVak** [comment]
@Romex91 I'm trying out the approach with emoji and different questions to have the prediction table also. So far working fine, thanks 👍
